stats/matrixify.py

```diff
@@ -92,12 +89,25 @@ def get_framework(path: str, rule: Dict[str, Any]) -> str:
     # if lang == hcl:
     return s[s.index(lang)+1].strip()
 
+# Reads 'cwe_to_metacategory.yml' to construct a map to convert a CWE to a metacategory
+def create_metacategory_map(path: str) -> Dict[str, str]:
+    cwe_mc_map = {} # {cwe, metacategory}
+
+    with open(path, 'r') as mc_map_file:
+        mc_map = yaml.safe_load(mc_map_file)
+
+        for mc in mc_map:
+            for cwe in mc_map[mc]:
+                cwe_mc_map[cwe] = mc
+
+    return cwe_mc_map
+
 def is_security(path: str) -> bool:
     return "security" in path
 
 def is_rule(path: str) -> bool:
     _, ext = os.path.splitext(path)
-    return "yaml" in ext or "yml" in ext
+    return ("yaml" in ext or "yml" in ext) and '/scripts/' not in path
```

Comment:
nit: Although unlikely, the parenthesized conditions could match extensions that are not explicitly "yaml" or "yml". Maybe consider the following
```suggestion
    return ext in (".yaml", ".yml") and '/scripts/' not in path
```

Reply:
Nice, thanks!